src/widgets/refinement-list/refinement-list.js

```diff
@@ -63,20 +67,22 @@ refinementList({
   [ autoHideContainer=true ],
   [ collapsible=false ],
   [ showMore.{templates: {active, inactive}, limit} ],
-  [ collapsible=false ]
+  [ collapsible=false ],
+  [ searchForValues.{placeholder, templates: {noResults}}],
 })`;
 function refinementList({
     container,
     attributeName,
     operator = 'or',
-    sortBy = ['count:desc', 'name:asc'],
+    sortBy = ['isRefined', 'count:desc', 'name:asc'],
```

Comment:
No it doesn't break, but it does change the behavior.

I had the impression that we discussed the defaults for react-instantsearch. And so I thought that we could move forward on this too. Anyway, if you use search for facet values you will add values that are not the most popular, so they won't be displayed in the list if isRefined is not first.